climada/hazard/test/test_tc_tracks.py

```diff
@@ -1359,6 +1359,111 @@ def test_track_land_params(self):
             on_land,
         )
 
+    def test_compute_density_tracks(self):
+        """Test `compute_track_density` to ensure proper density count."""
+        # create track
+        track = xr.Dataset(
+            {
+                "time_step": ("time", np.timedelta64(1, "h") * np.arange(4)),
+                "max_sustained_wind": ("time", [10, 20, 30, 20]),
+                "central_pressure": ("time", [1, 1, 1, 1]),
+                "radius_max_wind": ("time", [1, 1, 1, 1]),
+                "environnmental_pressure": ("time", [1, 1, 1, 1]),
+                "basin": ("time", ["NA", "NA", "NA", "NA"]),
+            },
+            coords={
+                "time": ("time", pd.date_range("2025-01-01", periods=4, freq="12H")),
+                "lat": ("time", [-90, -90, -90, -90]),
+                "lon": ("time", [-179, -169, -159, -149]),
+            },
+            attrs={
+                "max_sustained_wind_unit": "m/s",
+                "central_pressure_unit": "hPa",
+                "name": "storm_0",
+                "sid": "0",
+                "orig_event_flag": True,
+                "data_provider": "FAST",
+                "id_no": "0",
+                "category": "1",
+            },
+        )
+
+        tc_tracks = tc.TCTracks([track])
+
+        hist_abs, *_ = tc.compute_track_density(
+            tc_tracks,
+            res=10,
+            norm=None,
+        )
+        # hist_norm, *_ = tc.compute_track_density(tc_tracks, res=10, density=True)
+        hist_wind_min, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=11, wind_max=None
+        )
+        hist_wind_max, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=None, wind_max=30
+        )
+        hist_wind_max, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=None, wind_max=30
+        )
+        hist_wind_both, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=11, wind_max=29
+        )
+        self.assertEqual(hist_abs.shape, (17, 35))
+        self.assertEqual(hist_abs.sum(), 4)
+        self.assertEqual(hist_wind_min.sum(), 3)
+        self.assertEqual(hist_wind_max.sum(), 4)
+        self.assertEqual(hist_wind_both.sum(), 2)
+        # the track defined above occupy positions 0 to 4
+        np.testing.assert_array_equal(hist_abs[0, 0:4], [1, 1, 1, 1])
+
+    def test_normalize_hist(self):
+        """test the correct normalization of a 2D matrix by grid cell area and sum of
+        values of the matrix."""
+
+        M = np.ones((10, 10))
+        M_norm = tc.normalize_hist(res=10, hist_count=M, norm="sum")
```

Comment:
What is tc? This variable is not defined here... Also, what about the other possible values of the attributes of `nomarlize_hist`?